contracts/contracts/NoLossDao.sol

```diff
@@ -91,11 +91,10 @@ contract NoLossDao is Initializable {
     );
     _;
   }
-
+  // Work on this modifier to be more accurate
   modifier userHasNoActiveProposal(address givenAddress) {
```

Comment:
The title of this modifier somehow implies that a benefactor can have multiple proposals.

Maybe `benefactorsProposalIsInactive`.